### DISCUSSION TITLE
Updating after Installing from release

### DISCUSSION THREAD

**Discussion by gsgoldma** [ggml-org/llama.cpp#2393] [Q&A]
If I want to update, do I need to download the latest release- or can I use git pull if I installed it in a cloned repository?

**Comment by ianscrivener** [comment]
`git pull` - I have a simple bash script `update.sh` that pulls and rebuilds the code;
```
git pull
make clean
LLAMA_METAL=1 make -j
```
[ I am on MacOS.]
